pkg/cli/client/client.go

```diff
@@ -509,7 +509,26 @@ func isCosignSigned(ctx context.Context, repo, digestStr string, searchConf Sear
 	_, err := makeGETRequest(ctx, URL, username, password, searchConf.VerifyTLS,
 		searchConf.Debug, &result, searchConf.ResultWriter)
 
-	return err == nil
+	if err == nil {
+		return true
+	}
+
+	var referrers ispec.Index
+
+	URL = fmt.Sprintf("%s/v2/%s/referrers/%s?artifactType=%s",
```

Comment:
Use url.QueryEscape(str) from net/url package to compute artyfactType query parameter and then add to fmt.Sprintf. It is safer and also escapes "/" from "application/vnd.dev.cosign.artifact.sig.v1+json" string